http_proxy.go

```diff
@@ -65,6 +66,7 @@ type HTTPProxyConfig struct {
 	RequestModifiers  []martian.RequestModifier  `json:"-"`
 	ResponseModifiers []martian.ResponseModifier `json:"-"`
 	CloseAfterReply   bool                       `json:"close_after_reply"`
+	HeaderRemover     []string                   `json:"header_remover"`
```

Comment:
?

Reply:
changed to flag name i.e. `RemoveHeaders`